bundles/org.eclipse.equinox.p2.ui.sdk/META-INF/MANIFEST.MF

```diff
@@ -24,13 +23,10 @@ Import-Package: javax.xml.parsers,
  org.eclipse.equinox.p2.engine.query;version="[2.0.0,3.0.0)",
  org.eclipse.equinox.p2.metadata;version="[2.0.0,3.0.0)",
  org.eclipse.equinox.p2.operations;version="[2.0.0,3.0.0)",
- org.eclipse.equinox.p2.planner;version="2.0.0",
```

Comment:
> The method getProfileChangeRequest() from the type ProfileChangeOperation refers to the missing type IProfileChangeRequest

which is `org.eclipse.equinox.p2.planner.IProfileChangeRequest` from director,
see https://github.com/eclipse-equinox/p2/actions/runs/18962301156/job/54152074388?pr=932#step:6:3705